proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -105,8 +106,10 @@ message SearchResult {
 service SearchService {
   // Search searches the BSR.
   rpc Search(SearchRequest) returns (SearchResponse);
-  // SearchCommitReference searches in a repository
-  rpc SearchCommitReference(SearchCommitReferenceRequest) returns (SearchCommitReferenceResponse);
```

Comment:
Do we want to delete the existing endpoint? This will be a breaking change for the web, right?

Reply:
I added `SearchCommitReference` earlier this week, and it's not implemented nor depended on